src/cuttlefish_conf.erl

```diff
@@ -103,9 +103,9 @@ fold_conf_files(Filename, Conf0) ->
 
 expand_values(Filename, Conf) ->
     lists:map(fun({K, Value0}) ->
-                case re:split(Value0, "[$(<)]") of
-                    [_, _, _, IncludeFilename0, _] ->
-                        % this is a value of the format "$(<IncludeFilename), let's read the contents
+                case re:run(Value0, "^\\$\\(\\<(.*)\\)$", [{capture, all_but_first, list}]) of
```

Comment:
Should we make sure the captured string is non-empty? I mean, we could use `.+` instead of `.*` to avoid an empty filename.

Reply:
Yes, good idea, thanks @dumbbell. Pushed the commit.